lib/options.test.js

```diff
@@ -20,7 +20,7 @@ test('should set default databaseUrl for develpment', t => {
 
     t.true(
         options.databaseUrl ===
-            'postgres://unleash_user:passord@localhost:5432/unleash'
+            'postgres://unleash_user:password@localhost:5432/unleash'
```

Comment:
password is intentionally set to "passord" (which is password in Norwegian). Changing this will break developer getting started guide (and require everyone with a local dev setup of unleash to change their local password).

Reply:
Resolved